infrastructure/zk/src/compiler.ts

```diff
@@ -13,13 +13,24 @@ export async function compileSystemContracts() {
     await utils.spawn('yarn workspace system-contracts build');
 }
 
+export async function prepareCompilersCache() {
+    // TODO(tomek) remove line below as soon as there is at least one pushed docker image
+    await utils.spawn('zk docker build compilers');
+    await utils.spawn('docker create --name temp matterlabs/compilers:latest2.0');
+    await utils.spawn('mkdir -p /root/.cache/hardhat-nodejs/compilers-v2');
+    await utils.spawn('docker cp temp:./.cache/hardhat-nodejs/compilers-v2/ /root/.cache/hardhat-nodejs/');
+    await utils.spawn('chmod -R +x /root/.cache/hardhat-nodejs/');
+    await utils.spawn('docker rm temp');
+}
```

Comment:
This seems very environment-specific. Also, can we at least reuse whatever hardhat is using to derive this directories? `env-paths` AFAIR